src/middleware/navigation/has.overseas.entity.middleware.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response, NextFunction } from 'express';
+
+import { logger } from '../../utils/logger';
+import { SECURE_UPDATE_FILTER_URL } from '../../config';
+import { getApplicationData } from "../../utils/application.data";
+import { checkOverseasEntityDetailsEntered, NavigationErrorMessage } from './check.condition';
+
+export const hasOverseasEntity = (req: Request, res: Response, next: NextFunction): void => {
+  try {
+    if ( !checkOverseasEntityDetailsEntered(getApplicationData(req.session)) ) {
```

Comment:
Seems like personal preference but I think the project generally doesnt have spaces inside the if condition, so it would be like this: `if (!checkOverseasEntityDetailsEntered(getApplicationData(req.session))) {`